src/main/java/Display.java

```diff
@@ -0,0 +1,48 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Display {
+    public void printGreeting() {
+        System.out.println("Welcome to TicTacToe!");
+    }
+
+    public void promptPlayer(String playerMark) {
+        System.out.println(String.format("Make a move: %s", playerMark));
+    }
+
+    private List currentGridCells(Board board, int size) {
```

Comment:
There's a lot going on in this method. Should all this be the Display's responsibility?

Reply:
Moved to `Board`